src/NLSModels.jl

```diff
@@ -1,5 +1,5 @@
 export AbstractNLSModel, nls_meta, NLSCounters, reset!,
-       residual, residual!, jac_residual, jac_structure_residual, jac_coord_residual!,
+       residual, residual!, jac_residual, jac_structure_residual!, jac_structure_residual, jac_coord_residual!,
```

Comment:
Would it be too much asking to split this into a separate PR to keep things contained?

Reply:
Absolutely. I plan to separate this and some other small things, but I made the PR to test.